src/osvr/Util/LogRegistry.cpp

```diff
@@ -187,7 +205,7 @@ namespace util {
             // File sink - rotates daily
             std::string logDir;
             try {
-                size_t q_size = 1048576; // queue size must be power of 2
+                size_t q_size = 65536; // queue size must be power of 2
```

Comment:
I'd err on the side of being more synchronous than less, since we were originally very synchronous (every OSVR_DEV_VERBOSE had an endl with its implied flush) and logging is typically most important when things are working badly.